### ISSUE TITLE
examples within request bodies with properties named "schemas" are rewritten to an object

### ISSUE THREAD

**Issue by fantapop** [thim81/openapi-format#120]
Hi!  This is very similar to a bug I filed 9 months ago. https://github.com/thim81/openapi-format/issues/91   Almost all instances of the bug were fixed at that time but I believe that bug can happen wherever an openapi "example" can be added and the previous fix was focused only on examples under components.  I noticed a couple other instances of this in my spec for example:

```json
    "/api/scim/v2/Groups/{id}": {
      "put": {
        "requestBody": {
          "required": true,
          "content": {
            "application/json": {
              "schema": {
                "type": "object",
                "properties": {
                  "schemas": {
                    "type": "array",
                    "items": {
                      "type": "string"
                    }
                  }
                },
                "example": {
                  "schemas": [
                        "urn:ietf:params:scim:schemas:core:2.0:Group"
                    ]
```

rewrites the example to:
```json
        "example": {
          "schemas": [
            {
              "0": "u",
              "1": "r",
              "2": "n",
              "3": ":",
              "4": "i",
              "5": "e",
              "6": "t",
              "7": "f",
              "8": ":",
              "9": "p",
              "10": "a",
              "11": "r",
              "12": "a",
              "13": "m",
              "14": "s",
              "15": ":",
              "16": "s",
              "17": "c",
              "18": "i",
              "19": "m",
              "20": ":",
              "21": "s",
              "22": "c",
              "23": "h",
              "24": "e",
              "25": "m",
              "26": "a",
              "27": "s",
              "28": ":",
              "29": "c",
              "30": "o",
              "31": "r",
              "32": "e",
              "33": ":",
              "34": "2",
              "35": ".",
              "36": "0",
              "37": ":",
              "38": "G",
              "39": "r",
              "40": "o",
              "41": "u",
              "42": "p"
            }
          ]
        },
```

For my spec, the only paths I still see this bug are under a requestBody but you might want to consider a fix which includes anything with a parent of example or a grandparent of examples.    https://swagger.io/docs/specification/adding-examples/

Thanks for taking a look!

**Comment by thim81** [comment]
@fantapop you are correct. I did a very specific patch. I ll expand the logic and try to exclude the examples from this quirky conversion.

**Comment by thim81** [comment]
@fantapop The [PR](#122) will solve this case.

I tried to create other cases with example where this also could happen, but I was unable to get proper reproduction cases (even when asking AI for samples)

**Comment by fantapop** [comment]
Thanks for the quick turnaround. I'll see if I can test it out this evening.

**Comment by thim81** [comment]
hi @fantapop 

We just released version 1.21.2, with the fix included.

You can see it in action in the [openapi-format playground](https://openapi-format-playground.vercel.app/?input=H4sIAAAAAAAAA4VRwU4CMRC98xVNz7giJh56w4t6ERK4GQ%2B1DDKEbeu0ayRk%2F93pssjiFk023Zn33ry%2BSfcDIaTzYLVHqYS8LUbFWA4TinblGNpzzd0SgiH0EZ1NusUag%2BBPi6BLvwUxZY%2FJ7EmwgQgeDK7Q6KQuRGPHFp9AoR2%2F4WvurubPk9n8cbo4CiLGLSR63vfU3ktW1U00r%2BM6nLJdH8IfukRXsdMyQPBRQYj3brk7I1oKCZaMR6pg2OWMsxFs%2FDXCBIfZtutdb0Kz0rmCNcGsodQZJi26882e7m0DJsphX%2BGJH4UiQsg6%2FPhfojuXaCK9y9zRiDBCedmj4xIioX2XWVmdQftYnVkTvpqX%2Fn%2FHl3z8iqxCiCvlNekyqGCwVO2UMo5AjYuReiBX%2BVz0137IwV99tzvVx%2BrwT2c9qL8BIgwThVkDAAA%3D&config=H4sIAAAAAAAAA82VwU7DMAyGX2XKubzAjoBASEgcdqQcrNZtA0kcEgdtmvbupCvrkm7ShpBQj%2F7j2p%2FtJt4KT47Fkl3AQjRSMboVRkGIYn80GNvSLBalcERciuXitTd7gSwasLIUxUGRpqHE9Oi%2B0PlEscBdalekLRk0nIoMbWqub6Lw6CjYVMV1hDWg7qmKcq%2B%2BFQNniyeYDliSeapTtqA1uE2i1OgrJ23vmRE70Mh5GQ4%2FA3q%2BpXqTqT4W43HCY8nPDCjMjAe46mZFVKOK388KKY2QYJkoZhfwIkqfVTrMqqg61JDny%2BCShFcHrchwvNjTqGNxF2N2CHXesDHkqChpPiaNGp1igqN2fGaSvGenMjRjEvTQoYvQvLHZPBh19v65%2Fj9iianYkNOQVoVr0FZhNsoGguJzUNe08j%2Bp0mB%2FBfsNw9HHBP2DVJpd3GTSP%2BxX28s%2Bu78jpcB6rMWyAeWxd1jFZXd6POxFChxfzGcwbYAW4z5892TE7hvo6UDZPQcAAA%3D%3D)

If the issue is resolved, feel free to close this issue. If not, let me know.

**Comment by fantapop** [comment]
awesome, thank you. It's fixed.
